auditeval/test.go

```diff
@@ -264,3 +232,101 @@ func (t *testItem) evaluate(output string) (TestResult bool, ExpectedResult stri
 
 	return TestResult, ExpectedResult, err
 }
+
+func compareOp(tCompareOp string, flagVal string, tCompareValue string) (string, bool) {
```

Comment:
I think there will potentially be some merge issues because I've just seen a slightly different version of this change in another PR. I'm assuming the other one is the latest and not looking at this one in detail.

Reply:
I made the function signature the same, but yeah the body of the function will need a bit of tweaking to merge.